### PR TITLE
Feat: Add schema for Become a Parent Request

### PR DESCRIPTION
**Pull request by chadtomkiss-madetech**
**Work done**

- Added `Sequelize-cli` for generating and running migrations
- Added migration for `Approvals`

#### Tests
Does this PR include tests for the changes introduced?
- [ ] Yes
- [ ] No, I found it difficult to test
- [x] No, they are not required for this change

### CODE REVIEW

File: migrations/20200519170545-create_approvals_table.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        references: {
+          model: {
+            tableName: 'Establishment',
+            schema: 'cqc'
+          },
+          key: 'EstablishmentID'
+        },
+        allowNull: false
+      },
+      UserID: {
```
Comment (aaron-russell) [inline migrations/20200519170545-create_approvals_table.js:17]: Should we match the column name here?

File: migrations/20200519170545-create_approvals_table.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Approvals', {
+      EtablishmentID: {
```
Comment (aaron-russell) [inline migrations/20200519170545-create_approvals_table.js:6]: You've spelled Establishment wrong

File: server/config/sequelize.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+module.exports = {
+  development: {
+    username: appConfig.get('db.username'),
+    password: appConfig.get('db.password'),
+    database: appConfig.get('db.database'),
+    host: appConfig.get('db.host'),
+    port: appConfig.get('db.port'),
+    dialect: appConfig.get('db.dialect'),
+    migrationStorageTableSchema: 'cqc',
```
Comment (aaron-russell) [inline server/config/sequelize.js:11]: Can this be set by appConfig too?

File: package.json [hunk 4]
```diff
@@ -76,6 +76,7 @@
     "rxjs": "^6.5.2",
     "rxjs-compat": "^6.5.2",
     "sequelize": "^4.44.3",
+    "sequelize-cli": "^5.5.1",
```
Comment (aaron-russell) [inline package.json:79]: Is the version mismatch between the cli and sequelize going to cause an issue?

Comment (chadtomkiss-madetech) [reply package.json:79]: `sequelize --help` returns Sequelize CLI [Node: 13.8.0, CLI: 5.5.1, ORM: 4.44.3]

When creating a migration, you get:

```
up: (queryInterface, Sequelize)
```

The `Sequelize` argument that's passed is our installed 4.44.3 version - so as long as any Sequelize methods used in a migration are available in the version we use, we're good.

File: server/config/sequelize.js (hunk 3 above)
Comment (chadtomkiss-madetech) [reply server/config/sequelize.js:11]: This isn't defined in `appConfig`, and is also hardcoded in the models that are setup.

**Pull request merged**